### DISCUSSION TITLE
NDK support

### DISCUSSION THREAD

**Discussion by dhaigit** [tyron12233/CodeAssist#202] [General]
Does app support (or plan to support) NDK in building projects? I have some NDK projects that I would really like to be able to build with this.

**Comment by tyron12233** [comment]
There are currently no plans to add the NDK, as porting the toolchain is not an easy task not to mention the language code completion support for C/C++. The current plans are to improve support for Java, Kotlin, XML and Layout Preview.

Also next time to better give focus on `issues` of the current state of the app, Please use the [discussions tab](https://github.com/tyron12233/CodeAssist/discussions) for these kind of discussions.

**Comment by dhaigit** [comment]
A little sad to hear it doesn't and won't have NDK support.

**Comment by ghost** [comment]
In the past, AIDE manage to build working NDK for Android. You can check the source code https://github.com/android-ide/aide_ndk
